importccl: add hint to line too long errors

otan opened the pull request:
Release note (sql change): Added a hint for "line too long"
errors when importing a backup with long lines.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/52432)
<!-- Reviewable:end -->

otan committed:
importccl: add hint to line too long errors

Release note (sql change): Added a hint for "line too long"
errors when importing a backup with long lines.

pbardea reviewed (approved):
Nice - thanks!

otan commented:
bors r=pbardea

otan commented:
bors r=pbardea
